app/src/main/java/org/mozilla/fenix/search/toolbar/SearchSelectorMenu.kt

```diff
@@ -46,7 +48,15 @@ class SearchSelectorMenu(
         data class SearchEngine(val searchEngine: MozSearchEngine) : Item()
     }
 
-    val menuController: MenuController by lazy { BrowserMenuController() }
+    val menuController: MenuController by lazy {
+        BrowserMenuController(
+            style = MenuStyle(
+                horizontalOffset = (-4).dpToPx(context.resources.displayMetrics),
+                verticalOffset = (-2).dpToPx(context.resources.displayMetrics),
```

Comment:
maths behind the values: SelectorMenu has the height of 28 while its container is 40 (vertical spacing is 6, target is 4) and margintStart of 8, target 4 as well. Hence, -4, -2 adjustment

Reply:
Thanks for the breakdown - could we have this explanation as a comment? could save future effort in reverse engineering the maths